### PR TITLE
`borrow loan` - scale interest with amount of loan

### LINKED ISSUE #1117: `Estimated fees` alignment
211020 v0 11 1 - iOS : Android - DEX - Remove LP - Estimated fee margin alignment

#### What UI/UX improvements would you like to be implemented: 
iOS / Android
• _Margin_ for `Estimated Fee` should be even

• Edit english translation of **Title** to the following for consistency: 
`Estimated Fee = Estimated fee`

#### Why is this needed:
• _Margin_ for `Estimated Fee` is uneven
• Regarding suggestion of having lowercase `fee`, on Remove LP screen _fee_ is in **lowercase**, but in pop-up it's in **Sentence case**. Hence, suggestion for consistency. 

#### Which component/screens does this concern?:
• Send
• Convert DFI
• Add / Remove Liquidity
• Swap

### LINKED ISSUE #1637: Change `Total Interest Amount` to `Total Annual Interest` on Borrow Loan screen for clarity
#### What UI/UX improvements would you like to be implemented: 

To change label `Total Interest Amount` to `Estimated Annual Interest`. 

Currently, `Total Interest Amount` can be misleading because users might think this is the immediate interest he/she has to pay, which can impact the amount of tokens he/she chooses to mint, especially in situations where a large loan amount is involved.

telegram-cloud-photo-size-5-6165686520372112978-y

### LINKED ISSUE #1745: `borrow loan` - interest amount does not scale with loan amount to borrow
#### What happened:
- The value displayed for `Total loan + interest` does not scale with the loan amount to borrow

#### What you expected to happen:
- The value shown should scale with interest per block

#### How to reproduce it (as minimally and precisely as possible):
- Create a vault with 1 DFI collateral
- Browse loan token > choose DUSD
- Input `1` as the amount to borrow, note the value of `Total interest amount` and `Total loan + interest`
- Change the input to `5`, note value of of `Total interest amount` remain the same

#### Anything else we need to know?:
- Affecting borrow loan token screen and borrow more screen
- Does not affect calculation of `Resulting collateralization ratio`

### PR DESCRIPTION
**Pull request by kyleleow**
#### What kind of PR is this?:

/kind fix

#### Which issue(s) does this PR fixes?:

Fixes #1745
Fixes #1117 
Fixes #1637 

#### Developer Checklist:

- [x] Read your code changes at least once
- [x] Tested on iOS/Android device (e.g, No crashes, library supported etc.)
- [x] No console errors on web
- [x] Tested on Light mode and Dark mode*
- [x] Your UI implementation visually matched the rendered design*
- [ ] Unit tests*
- [ ] Added e2e tests*
- [ ] Added translations*

### CODE REVIEW

File: mobile-app/app/screens/AppNavigator/screens/Loans/screens/BorrowLoanTokenScreen.tsx [hunk 1]
```diff
@@ -72,6 +75,8 @@ export function BorrowLoanTokenScreen ({
     new BigNumber(loanToken.activePrice?.active?.amount ?? 0),
     interestPerBlock
   )
+  const { network } = useNetworkContext()
+  const blocksPerDay = network === EnvironmentNetwork.MainNet || network === EnvironmentNetwork.TestNet ? 2880 : 144
```
Comment (thedoublejay) [inline mobile-app/app/screens/AppNavigator/screens/Loans/screens/BorrowLoanTokenScreen.tsx:79]: I think this one is used somewhere else right? Maybe we can replace with hooks?

File: mobile-app/app/screens/AppNavigator/screens/Loans/screens/BorrowMoreScreen.tsx [hunk 2]
```diff
@@ -70,8 +75,9 @@ export function BorrowMoreScreen ({ route, navigation }: Props): JSX.Element {
     if (vault === undefined || amountToAdd === undefined || loanToken?.activePrice?.active?.amount === undefined) {
       return
     }
-
-    setTotalLoanWithInterest(new BigNumber(amountToAdd.amountInToken).plus(interestPerBlock))
+    const annualInterest = interestPerBlock.multipliedBy(blocksPerDay * 365).multipliedBy(amountToAdd.amountInToken)
```
Comment (thedoublejay) [inline mobile-app/app/screens/AppNavigator/screens/Loans/screens/BorrowMoreScreen.tsx:78]: same here, we can update to hook

**Review by thedoublejay — CHANGES_REQUESTED** [review]

File: mobile-app/app/screens/AppNavigator/screens/Loans/screens/BorrowLoanTokenScreen.tsx (hunk 1 above)
Comment (kyleleow) [reply mobile-app/app/screens/AppNavigator/screens/Loans/screens/BorrowLoanTokenScreen.tsx]: updated

File: mobile-app/app/screens/AppNavigator/screens/Loans/screens/BorrowMoreScreen.tsx (hunk 2 above)
Comment (kyleleow) [reply mobile-app/app/screens/AppNavigator/screens/Loans/screens/BorrowMoreScreen.tsx:76]: updated `BorrowMoreScreen`

**Review by thedoublejay — APPROVED** [review]

**Pull request merged**